homeassistant/components/switchbee/light.py

```diff
@@ -0,0 +1,175 @@
+"""Support for SwitchBee light."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any, cast
+
+from switchbee.api import SwitchBeeDeviceOfflineError, SwitchBeeError
+from switchbee.device import ApiStateCommand, DeviceType, SwitchBeeDimmer
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, ColorMode, LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import SwitchBeeCoordinator
+from .entity import SwitchBeeDeviceEntity
+
+MAX_BRIGHTNESS = 255
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def _brightness_hass_to_switchbee(value: int) -> int:
+    """Convert hass brightness to SwitchBee."""
+    sb_brightness = int(100 * (value) / MAX_BRIGHTNESS)
+    # SwitchBee maximum brightness is 99
+    return sb_brightness if sb_brightness != 100 else 99
+
+
+def _brightness_switchbee_to_hass(value: int) -> int:
```

Comment:
I think it is preferred to have "sentences":
```suggestion
def _switchbee_brightness_to_hass(value: int) -> int:
```